geoviews/element/geo.py

```diff
@@ -209,10 +209,20 @@ def range(self, dim, data_range=True, dimension_range=True):
 
 class WMTS(_GeoFeature):
     """
-    The WMTS Element represents a Web Map Tile Service specified as a
-    URL containing {x}, {y}, and {z} templating variables, e.g.:
+    The WMTS Element represents a Web Map Tile Service specified as URL
+    containing different template variables or xyzservices.TileProvider.
 
-    https://maps.wikimedia.org/osm-intl/{Z}/{X}/{Y}@2x.png
+    These variables correspond to three different formats for specifying the spatial
+    location and zoom level of the requested tiles:
+
+    * Web mapping tiles sources containing {x}, {y}, and {z} variables
```

Comment:
I'm a little confused by this change. You are changing the `WMTS` class to contain `WMTS`,  `QUADKEY`, and `BBox`.

Reply:
Technically, it already supported all those; it just wasn't documented. https://github.com/holoviz/geoviews/blob/ec245c7ddd58ed9afc2430f6f5675255b61ba247/geoviews/plotting/bokeh/__init__.py#L52-L56

Reply:
https://github.com/holoviz/geoviews/blob/ec245c7ddd58ed9afc2430f6f5675255b61ba247/geoviews/plotting/mpl/__init__.py#L474-L511

Reply:
> Technically, it already supported all those; it just wasn't documented. 

Isn't this something you added to this PR?

It just doesn't make sense to me to have a specific name for the class and then support more general cases.

Reply:
No, this PR only adds support for XYZServices' classes (the package).

Yes, I understand that, but it'd be breaking change if we separated those out.

Reply:
For example, this URL uses `BBoxTileSource` and all I'm using is `gv.WMTS`
https://ahuang11.github.io/web-map-feature-services-cookbook/notebooks/web_map_services.html#accessing-external-sources

Reply:
I think I got confused by the way Github highlighted stuff. 

I think we should think about giving it a broader name for the class (but not in this PR).

Reply:
> Maybe we can deprecate `gv.WMTS` in favor of `gv.Tiles`

I think this would be a good idea. Let's do it in another PR.